code/bit_manipulation/src/count_set_bits/brian_Kernighan_Algorithm.cpp

```diff
@@ -0,0 +1,29 @@
+//Brian Kernighan’s Algorithm. This programme uses O(logn) to count set bits.
+//Part of Cosmos by OpenGenus Foundation 
+#include <iostream> 
+using namespace std; 
```

Comment:
Don't use this. Fix your filename to be all lowercase and move the algorithm to its own folder.